pkg/chi-middleware/oapi_validate.go

```diff
@@ -59,11 +59,13 @@ func OapiRequestValidatorWithOptions(swagger *openapi3.T, options *Options) func
 			// validate request
 			if statusCode, err := validateRequest(r, router, options); err != nil {
 				if options != nil && options.ErrorHandler != nil {
-					options.ErrorHandler(w, err.Error(), statusCode)
+					options.ErrorHandler(w, r, err.Error(), statusCode)
 				} else {
 					http.Error(w, err.Error(), statusCode)
 				}
-				return
+				// FIXME: [ENG-1248] Remove bring back the return statement
+				// return
+				next.ServeHTTP(w, r)
```

Comment:
You either need to remove this ServerHTTP call or still return
```suggestion
```

or 
```suggestion
				next.ServeHTTP(w, r)
				return
```